### PR TITLE
🌍 Update README.md

### PR DESCRIPTION
**Pull request by jeffzh4ng**
## Purpose
Update README.md

- add image
- add client and server readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,4 +13,7 @@
 1. run `npm run dev` to start client
 
 ## Documentation
-- [Server architecture](server/ARCHITECTURE.md)
+
+- [Client README.md](client/README.md)
+- [Server README.md](server/README.md)
+- [Server ARCHITECTURE.md](server/ARCHITECTURE.md)
```
Comment (xujustinj) [inline README.md:19]: nit: is the `.md` important to include here?

**Review by xujustinj — APPROVED** [review]
a-ok, assuming the banner is creative commons... (it is, right?)

File: README.md (hunk 1 above)
Comment (jeffzh4ng) [reply README.md]: agreed, and changed.

**Comment by jeffzh4ng** [comment]
> a-ok, assuming the banner is creative commons... (it is, right?)

It's some random anime picture from google 😅
Worst case scenario, they just ask us to take it down. There's no company here to sue?

**Review by xujustinj — COMMENTED** [review]
same intellectual property concern stands

**Review by xujustinj — APPROVED** [review]

**Pull request merged**